commands/explain.js

```diff
@@ -0,0 +1,8 @@
+const game = require('../data/cards');
+const { printCard } = require('../util/format');
+const cards = Object.values(game).flat(2);
+
+module.exports = (character) => {
+  character = new RegExp(character, 'i');
+  printCard(cards.filter(card => card.name.match(character))[0]);
```

Comment:
Why take only the first card? If there are multiple matches, we should print all of them.
If you only want the first card, use `Array.prototype.find()` instead.